src/javascript/_common/base/client_base.js

```diff
@@ -381,7 +381,7 @@ const ClientBase = (() => {
         // or for other clients if current account is cryptocurrency it should only transfer to fiat currencies and vice versa
         const is_from_crypto = isCryptocurrency(from_currency);
         const is_to_crypto   = isCryptocurrency(to_currency);
-        return (is_from_crypto ? !is_to_crypto : is_to_crypto);
+        return { is_from_crypto, is_to_crypto };
```

Comment:
i believe we need to return a boolean here

Reply:
with boolean only transfer crypto to crypto is enabled, can't make fiat to crypto transfer